admin/assets/js/filebrowser/assets/css/filebrowser.css

```diff
@@ -144,15 +144,15 @@
 
 #actionwindow-wrapper {
   position: absolute;
-  margin: 60px 30%;
+  margin: 30px 30px;
```

Comment:
I would revert this change;  the old css makes sure that the popup is always centered in the middle.

Reply:
@jeroenweareorangeeu Reverted, cheers.